doc/source/workflows/basics.rst

```diff
@@ -32,11 +33,22 @@ Here is a single three-node DAG:
     def aggregate(data: List[float]) -> float:
         return sum(data)
 
-    # Build the DAG.
+    # Build the DAG:
+    # data -> preprocessed_data -> aggregate
     data = read_data.bind(10)
     preprocessed_data = preprocessing.bind(data)
     output = aggregate.bind(preprocessed_data)
 
```

Comment:
Break the example here. The plotting of the DAG is not part of its definition. I think it may be better to first show how you execute the DAG to get functionality as before and then elaborate on the differences including the ability to visualize the DAG.